plugin/src/sidebar.ts

```diff
@@ -14,10 +14,8 @@ let addedDatasets: string | undefined = undefined;
 
 let rawCatalog: CatalogRawItem[] = [];
 
-const doNotShowWelcome = true; // Make it `let doNotShowWelcome: boolean = false`, and then modify based on storage value when Storage API available
-
 // let isMobile: boolean;
-
+let welcomePagIsOpen = false;
```

Comment:
welcomePageIsOpen
Missing the `e` on Page

Reply:
done

Reply:
a36d7887f9fae43211d25e5df9545ce613bbccec